API/admin/reporting/util/countSkillProfiles.js

```diff
@@ -0,0 +1,29 @@
+const Sequelize = require("sequelize");
+const Models = require("../../../../models");
+const Skills = Models.skill; // Skills Table
+const Profiles = Models.profile; // Profiles Table
+
+const countSkillProfiles = async () => {
+  const profileSkills = await Skills.findAll({
+    // {Skill ID (id) , descriptionEn, descriptionFr, Count Profile Occurences (countOccurences)}
```

Comment:
Is this comment necessary ?

Reply:
It's just so someone knows how the object is getting returned.